dm/tests/safe_mode/run.sh

```diff
@@ -211,6 +211,8 @@ function safe_mode_duration() {
 	run_sql_file $cur/data/db1.increment.sql $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1
 	run_sql_file $cur/data/db2.increment.sql $MYSQL_HOST2 $MYSQL_PORT2 $MYSQL_PASSWORD2
 
+	# make sure worker1 enter to sync status, and complete one dml
+	check_log_contain_with_retry "alter table t1 add column age int" $WORK_DIR/worker1/log/dm-worker.log
```

Comment:
The code change doesn't match the PR title. So which one should be changed?

Reply:
it is working in progress. I will update it